sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Diagnostics/DiagnosticsTests.cs

```diff
@@ -68,90 +68,93 @@ public async Task CheckpointManagerCreatesScope()
         }
 
         /// <summary>
-        ///   Verifies diagnostics functionality of the <see cref="PartitionPump" />
-        ///   class.
+        ///   Verifies diagnostics functionality of the <see cref="EventProcessorClient.RunPartitionProcessingAsync" />
+        ///   method.
         /// </summary>
         ///
         [Test]
-        [Ignore("Needs to be updated because Partition Pump class does not exist anymore.")]
-        public async Task PartitionPumpCreatesScopeForEventProcessing()
+        [Ignore("Diagnostic scope is not completing properly. Maybe the listener is being disposed of first.")]
+        public async Task RunPartitionProcessingAsyncCreatesScopeForEventProcessing()
```

Comment:
The test has been updated and all assertions pass. However, when disposing of the Diagnostic Listener, it throws an exception because the processor scope is not completed. Still couldn't figure out what the problem is, but it might be related to the test infrastructure.

Reply:
Thanks for the context.   I think you're doing the right thing in leaving the `Ignore` in place.    Would you be so kind as to open a tracking issue for us to revisit?   We've got some diagnostics work coming up, so that'd probably be a good place to revisit this.

Reply:
Issue opened: #9225